Sources/WebDriver/Session.swift

```diff
@@ -321,6 +328,14 @@ public class Session {
         try webDriver.send(Requests.SessionWindowSize.Post(session: id, windowHandle: handle, width: width, height: height))
     }
 
+    /// - Parameters:
+    ///   - latitude: Number coordinate of current geo location
```

Comment:
These comments are not very useful. Better only have `/// Set the current geo location.`